<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeMed</class>
 <widget class="QDialog" name="ChangeMed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change medicine</string>
  </property>
  <property name="styleSheet">
   <string notr="true">ChangeMed {
	background-color: rgb(255, 255, 255);
}


QPushButton {
 background-color: red;
 color: white;
 padding: 10px 20px;
 text-align: center;
 font-size: 16px;
 margin: 4px 2px;
 border-radius: 4;
 font: 57 8pt &quot;TT Norms Medium&quot;;
}

QPushButton:hover {
     background-color: #FF5757;
}

QPushButton:pressed {
  background-color: #3e8e41;
}

QTextBrowser {
	background-color: white;
	color: red;
	border: none;
   border-radius: 4;
   font: 57 8pt &quot;TT Norms Light&quot;;
}

QLabel {
 font: 57 8pt &quot;TT Norms Medium&quot;;
text-align: center;
color: red;
}

QComboBox {
background-color: rgb(255, 255, 255);
 font: 57 8pt &quot;TT Norms Medium&quot;;
border: none;
}

QSpinBox {
background-color: rgb(255, 255, 255);
 font: 57 8pt &quot;TT Norms Medium&quot;;
border: none;
}

QLineEdit {
 font: 57 8pt &quot;TT Norms Medium&quot;;
}
</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Choose medicine to change:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxName"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Start Date: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditStartDate"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Frequency:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxFreq"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Amount:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxAmount"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Time:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxTime"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Comment:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditCommetn"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonDelete">
      <property name="text">
       <string>Delete medicine</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Save all</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
